service/index.js

```diff
@@ -20,6 +20,11 @@ var one_week = one_day * 7;
 var one_year = one_day * 365;
 var contentTypes = {".js": 'application/javascript', ".css": 'text/css'};
 
+app.get(/^\/v([12])\/robots\.txt$/, function (req, res) {
+    res.type('text/plain');
+    res.send("User-agent: *\nDisallow: /");
```

Comment:
This will disallow documentation pages, which we want to allow to be scraped.  Also I'm not totally convinced that deploying robots.txt in a subfolder will work for all robots.  I'd suggest publishing it at the root but specifying the specific path patterns that we want to exclude.  Can you do path patterns in robots?